README.md

```diff
@@ -14,42 +14,82 @@ We use this prototype internally to LeanIX to generate SBOMs for our projects. I
            -e LEANIX_TOKEN='<leanix-technical-user-token>' \
+           -e GIT_PROVIDER='GITHUB' \
            -e GITHUB_TOKEN='<github-token>' \
            -e GITHUB_ORGANIZATION='<github-organization>' \
-           leanixacrpublic.azurecr.io/vsm-sbom-booster -rm
+           leanixacrpublic.azurecr.io/vsm-sbom-booster
 ```
+
+GitLab:
+```console
+docker run --pull=always --rm \
+           -v /var/run/docker.sock:/var/run/docker.sock \
+           -v <temp-folder-to-be-used-for-storing-data>:/tempDir \
+           -e MOUNTED_VOLUME='<temp-folder-to-be-used-for-storing-data>' \
+           -e LEANIX_HOST='<leanix-workspace-host>' \
+           -e LEANIX_REGION='<leanix-workspace-region>' \
+           -e LEANIX_TOKEN='<leanix-technical-user-token>' \
+           -e GIT_PROVIDER='GITLAB' \
+           -e GITLAB_TOKEN='<github-token>' \
+           -e GITLAB_GROUP='<github-organization>' \
+           leanixacrpublic.azurecr.io/vsm-sbom-booster
+```
+
 2. After a while your mapping inbox should be receiving new discovery items. These will need to be mapped by you once (see our [user documentation](https://docs-vsm.leanix.net/docs/discover-automate#create-your-service-baseline)).
 
 ### Environment Variables
 
+#### Runtime settings
 The first `-v` param is needed as the setup is a docker-in-docker setup and will need to mount the local docker runtime into the container. Under normal circumstances this param can be copied and pasted.
 
 The second `-v` param is the path to temporary folder that the `vsm-sbom-booster`will use to temporarily clone the projects to attempt to generate the SBOM. e.g. `~/output/temp`. Docker should have Read/Write access on this folder.
 
 `MOUNTED_VOLUME`: this is the same as the second `-v` param. It's required as the container needs an explicit env variable to do its job.
 
+`CONCURRENCY_FACTOR`(optional): the number of parallel jobs `vsm-sbom-booster` will use to generate SBOMs. Note: increasing this number will come at higher compute costs. Default: 3
+
+#### LeanIX configs
+
+`LEANIX_HOST`: The host name of your LeanIX workspace. e.g. For `https://acme.leanix.net` you would provide `acme`
+
+`LEANIX_REGION`: The region that your VSM workspace is hosted in. One of: eu|de|us|au|ca|ch
+
 `LEANIX_TOKEN`: API token with **ADMIN** rights from your VSM workspace. (see admin > technical users).
 
+#### Discovery API data
+
+`SOURCE_TYPE` (optional): this will the source system from where you're scanning. This is used in the mapping inbox to understand where discovered data originated from. Default: `vsm-sbom-booster`
+
+`SOURCE_INSTANCE`(optional): individual instance within the source system e.g. prod or org entity within the source system. This is used in the mapping inbox to understand where discovered data originated from. Default: GitHub Org or GitLab Group + Sub Groups
+
+#### GIT
+
+`GIT_PPROVIDER`: This is the Git provider to scan and generate SBOMs from. For now, either `GITHUB` or `GITLAB`.
```

Comment:
could you add a note pls that for GHE: you would need to put `GITHUB` and insert the API (i suppose?)

Reply:
I feel like the values are self explanatory, and the definition of the GITHUB_API_HOST mentions that you need it for GHE. Do you believe that it needs more still?

Reply:
i trust your assessment from the field. Lets head out like this. It can always be changed if demand is there